packages/components/src/toolbar/toolbar-item/types.ts

```diff
@@ -0,0 +1,21 @@
+/**
+ * External dependencies
+ */
+import type { ElementType } from 'react';
+
+export type ToolbarItemProps = {
```

Comment:
We could easily forward the types from reakit/Toolbar/ToolbarItem as well. Props is covered a lot of space here.

Reply:
I think so! We can straight up do:

```tsx
function ToolbarItem(
	{
		children,
		as: Component,
		...props
	}: React.ComponentPropsWithoutRef< typeof BaseToolbarItem >,
```